public/request/game/update-rich-presence.php

```diff
@@ -17,7 +17,7 @@
 $gameId = (int) $input['game'];
 
 // Only allow jr. devs if they are the sole author of the set or have the primary claim
-if ($permissions === Permissions::JuniorDeveloper && (!checkIfSoleDeveloper($user, $gameId) && !hasSetClaimed($user, $gameId, true, ClaimSetType::NewSet))) {
+if ($permissions === Permissions::JuniorDeveloper && (checkIfSoleDeveloper($user, $gameId) || hasSetClaimed($user, $gameId, true, ClaimSetType::NewSet))) {
```

Comment:
This logic was correct before. If the logic evaluates true, an error is generated. So you have to check for the inverse to allow the functionality. You're effectively making it so Junior Developers can only edit sets they aren't the sole developer for, or don't have claimed.

Reply:
My mistake, this line change has been reverted in https://github.com/RetroAchievements/RAWeb/pull/1494/commits/2251585fb498509f776f598f418e37ed59bf5896.